src/system/SystemConfig.h

```diff
@@ -690,6 +724,20 @@ struct LwIPEvent;
 #endif
 #endif // CHIP_SYSTEM_CONFIG_USE_ZEPHYR_NET_IF
 
+#ifndef CHIP_SYSTEM_CONFIG_USE_MBED_NET_IF
```

Comment:
Unused?

Reply:
Good catch, we used something similar to the Zephyr port at the beginning of the project but just use the POSIX API now. I remove.